### DISCUSSION TITLE
Is there a way to create a link to automatically open Session messenger and start a chat with a Session ID?

### DISCUSSION THREAD

**Discussion by BenK885** [oxen-io/session-desktop#3204] [Q&A]
Is there a way to create a link to automatically open Session messenger and start a chat with a Session ID?
Like "http://t.me/<username>" for Telegram or "https://wa.me/1XXXXXXXXXX" for WhatsApp?

What about the "sessionmessenger://" URL scheme on iOS?

By the way: Telegram uses "tg://resolve?domain=username"

**Comment by duhast** [comment]
+1 for custom URL scheme support

**Reply by hydrian** [reply]
probable better as something like this 

### for users:
session-chat://user/{userid}

### for community chat rooms
session-chat://community/{community id}
